test/jdk/java/awt/Mouse/MouseComboBoxTest/MouseComboBoxTest.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2014, Oracle and/or its affiliates. All rights reserved.
+ * Copyright (c) 2021, Oracle and/or its affiliates. All rights reserved.
```

Comment:
We should keep both initial year of addition and latest updated year. Please update it before pushing.